applications/GeoMechanicsApplication/custom_elements/transient_thermal_element.h

```diff
@@ -86,7 +91,8 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientThermalElement : public Ele
                                                                    GetIntegrationMethod());
         }
 
-        const auto integration_coefficients = CalculateIntegrationCoefficients(det_J_container);
+        const auto integration_coefficients = mIntegrationCoefficientsCalculator.Run<vector<double>>(
```

Comment:
For clarity, I would write this as follows:
```suggestion
        const auto integration_coefficients = mIntegrationCoefficientsCalculator.Run<Vector>(
```

Reply:
changed